Tester.java

```diff
@@ -1,3 +1,4 @@
+import java.util.Objects;
```

Comment:
I'm not gonna approve this until #7 is done since there's alot of changes that need doing. Once #7 and #6 are merged into main, main will be merged into this which should remove the diff for this file.